packages/utils/src/lib/file-write-runner-config.ts

```diff
@@ -0,0 +1,18 @@
+import { platform } from 'os';
+import { AuditOutput, RunnerConfig } from '@code-pushup/models';
+import { toUnixPath } from './utils';
+
+export function createFileWriteRunnerConfig(
+  output: AuditOutput[],
+  outputFilePath: string,
+): RunnerConfig {
+  const auditOutput =
+    platform() === 'win32'
+      ? JSON.stringify(output)
+      : "'" + JSON.stringify(output) + "'";
+  return {
+    command: 'echo',
+    args: [auditOutput, '>', toUnixPath(outputFilePath)],
+    outputFile: toUnixPath(outputFilePath),
```

Comment:
maybe [path.join](https://nodejs.org/api/path.html#pathjoinpaths) or [path.normalize](https://nodejs.org/api/path.html#pathnormalizepath) could be used instead

Reply:
I think just `outputFile: outputFilePath` should be fine, the file path should be platform-dependant.

The `toUnixPath` function is needed by the ESLint plugin, because it emits source file issues and needs to convert them absolute platform-dependant paths to relative Git paths. I'm not aware of any other use case.

Reply:
If you say so. I wanted to make sure that the resulting path would be safe but if that is the responsibility of the caller, sure.